assets/css/reset.css

```diff
@@ -0,0 +1,34 @@
+/* Resets the box size of every element */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+  }
+  
+  /* Gives the body element a parent height to compare against */
+  html {
+    height: 100%;
+  }
+  
+  body {
+    min-height: 100%;
+    line-height: 1;
+    font-family: sans-serif;
+  }
+  
+  h1, h2, h3, h4, h5, h6 {
+    font-size: 100%;
+  }
+  
+  /* Matches the font of special elements to the rest of the page */
+  input, select, option, optgroup, textarea, button,
+  pre, code {
+    font-size: 100%;
+    font-family: inherit;
+  }
+  
+  /* Removes default bullet points from lists */
+  ol, ul {
```

Comment:
I think rather than removing the default for all lists, we should make a class instead and apply the styling to the class. Then we can apply the class to the lists it applies to.